jobs/mongodb_migration/src/mongodb_migration/deletion_migrations.py

```diff
@@ -16,6 +16,11 @@
 
 
 class MetricsDeletionMigration(MetricsMigration):
+    def __init__(self, job_type: str, cache_kind: str, version: str, description: Optional[str] = None):
```

Comment:
I got tired from copy-pasting descriptions too :smile:  is this change okay? doesn't it affect old migrations (i'm not sure how description is used internally).

Reply:
yes, much better! thanks for all these improvements